internal/resources/resource_host.go

```diff
@@ -52,6 +53,23 @@ const (
 	allowedImageLength = 2
 )
 
+// HostStateEnum values. see hpe-hcss/quake/model/host.go.
```

Comment:
This comment might be confusing. The general public will be able to see this repo as it is open sourced. However they will not be able to see things under hpe-hcss.